### PR TITLE
Add Dependabot

### PR DESCRIPTION
**Pull request by larouxn**
Adds a [Dependabot](https://dependabot.com) config to help us keep our Python dependencies up to date. ✨

I have validated the config via [Dependabot's validation tool](https://dependabot.com/docs/config-file/validator/).
Screen Shot 2019-10-04 at 07 55 57

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "weekly"
+    default_labels:
+      - "dependencies"
+    default_reviewers:
+      - "Zlopez"
+      - "vfazio"
```
Comment (larouxn) [inline .dependabot/config.yml]: @Zlopez, @vfazio, are you two alright with being default reviewers? I sourced you two from the [most recent contributors](https://github.com/release-monitoring/anitya/graphs/contributors?from=2019-05-21&to=2019-10-04&type=c) GitHub page.

Alternatively or additionally, do you have suggestions for who should be default reviewers?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "weekly"
+    default_labels:
+      - "dependencies"
+    default_reviewers:
+      - "Zlopez"
+      - "jeremycline"
```
Comment (larouxn) [inline .dependabot/config.yml]: `default_reviewers` sourced from members of `release-monitoring`.
Screen Shot 2019-10-04 at 08 47 51

**Comment by Zlopez** [comment]
This sounds like an awesome addition, could you add news file for this PR, so we could merge it?

Look [here](https://anitya.readthedocs.io/en/stable/contributing.html#release-notes) to see, how is this done.

**Comment by larouxn** [comment]
I have added a `/news` release notes file and checked out the draft changelog.
Screen Shot 2019-10-07 at 18 18 41

**Comment by Zlopez** [comment]
Could you squash the commits?

**Comment by larouxn** [comment]
Sure thing 👍

**Comment by larouxn** [comment]
Note, I believe either you or @jeremycline will need to log into [Dependabot.com](https://dependabot.com) in order to grant it permission to this repository at some point.

**Comment by Zlopez** [comment]
Done

**Pull request merged**